lambdas/WavWrapLambdaFunction/index.js

```diff
@@ -115,7 +105,10 @@ exports.handler = async (event) => {
 
   // create the wav object
   const wav = new wavefile.WaveFile();
-  wav.fromMpeg(s3Object.Body);
+
+  if (s3Object.Body instanceof Uint8Array || Buffer.isBuffer(s3Object.Body)) {
```

Comment:
So what happens if this doesn't match?  Does that end up as an error somewhere?

Reply:
From what I've seen if `wav.fromMpeg` never gets called, `wavefile` will error out once you ask it to start doing things. But this could also `throw` here to make it more explicit

Reply:
+1 I'd vote for throwing something explicit.  Just in case the later error is ambiguous.